src/main/python/systemPython.ts

```diff
@@ -1,12 +1,18 @@
+import { LocalStorage } from 'node-localstorage';
 import { PythonInfo } from '../../common/common-types';
 import { getPythonVersion, isSupportedPythonVersion } from './version';
 
-export const getSystemPython = async (): Promise<PythonInfo> => {
-    for (const py of ['python', 'python3']) {
-        // eslint-disable-next-line no-await-in-loop
-        const version = await getPythonVersion(py).catch(() => null);
-        if (version && isSupportedPythonVersion(version)) {
-            return { python: py, version };
+export const getSystemPython = async (localStorageLocation: string): Promise<PythonInfo> => {
+    const localStorage = new LocalStorage(localStorageLocation);
+    const systemPythonLocation = localStorage.getItem('system-python-location');
```

Comment:
I don't like this API. Why does `getSystemPython` have to know how we store settings?

Instead, how about we pass in a string array of python locations to check? The calling code would look something like this:
```ts
    const localStorage = new LocalStorage(localStorageLocation);
    const systemPythonLocation: string | null = localStorage.getItem('system-python-location');
    pythonInfo = await getSystemPython(systemPythonLocation ? [systemPythonLocation] : []);
```

Reply:
That's a good idea